test/1_test_fullpath.js

```diff
@@ -1462,20 +1452,16 @@ contract('Voucher tests - UNHAPPY PATH', async (addresses) => {
       );
 
       //Check VoucherKernel state
-      const voucherStatusAfter = await contractVoucherKernel.vouchersStatus(
+      const voucherStatusAfter = await contractVoucherKernel.getVoucherStatus(
         tokenVoucherKey1
       );
 
       //Check it didn't go into a code branch that changes the complainPeriodStart
-      assert.isTrue(
-        voucherStatusAfter.complainPeriodStart.eq(
-          voucherStatusBefore.complainPeriodStart
-        )
-      );
+      assert.isTrue(voucherStatusAfter[3].eq(voucherStatusBefore[3]));
 
       // [1010.1100] = hex"AC" = 172 = REFUND_COMPLAIN_COF
       assert.equal(
-        web3.utils.toHex(voucherStatusAfter.status),
+        web3.utils.toHex(voucherStatusAfter[0]),
```

Comment:
Please use constants instead of hard-coded indexes for voucherStatusAfter (same as remark above)

Reply:
Remaining hard-coded indexes at lines 1531 (!! caution: 2 times) and 1535